doc/manual/rl-next/inherit-from-by-need.md

```diff
@@ -0,0 +1,7 @@
+---
+synopsis: "`inherit (x) ...` evalutates `x` only once"
+prs: 9847
+---
+
+`inherit (x) a b ...` now evalutates the expression `x` only once for all inherited attributes rather than once for each inherited attribute.
```

Comment:
```suggestion
synopsis: "`inherit (x) ...` evaluates `x` only once"
prs: 9847
---

`inherit (x) a b ...` now evaluates the expression `x` only once for all inherited attributes rather than once for each inherited attribute.
```